portal/src/main/webapp/js/src/oncoprint/studyview.js

```diff
@@ -34,6 +118,7 @@ $(document).ready(function() {
 			window.onc_obj = setUpOncoprint('oncoprint_body', {
 				sample_to_patient: window.PortalGlobals.getPatientSampleIdMap(),
 				gene_data: response.toJSON(),
+                                genepanel_data:genepanelValues,
```

Comment:
so what can we do here?


On Thu, Sep 17, 2015 at 1:23 PM, adamabeshouse <notifications@github.com>
wrote:

> In portal/src/main/webapp/js/src/oncoprint/studyview.js
> <https://github.com/cBioPortal/cbioportal/pull/254#discussion_r39773043>:
>
> > @@ -34,6 +118,7 @@ $(document).ready(function() {
> >  			window.onc_obj = setUpOncoprint('oncoprint_body', {
> >  				sample_to_patient: window.PortalGlobals.getPatientSampleIdMap(),
> >  				gene_data: response.toJSON(),
> > +                                genepanel_data:genepanelValues,
>
> we don't want to be passing this attribute
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cBioPortal/cbioportal/pull/254/files#r39773043>.
>